### PR TITLE
Minor Fixes in Browser/Editor Tool.

### PR DESCRIPTION
**Pull request by pankti-m**
- Rename CorfuStoreBrowser
- Eliminate time stamps from output of browser/editor tool
- Improve wording in README

## Overview

Description:

Why should this be merged: 

Related issue(s) (if applicable): #<number>

## Checklist (Definition of Done):

- [ ] There are no TODOs left in the code
- [ ] [Coding conventions](https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines) (e.g. for logging, unit tests) have been followed
- [ ] Change is covered by automated tests
- [ ] Public API has Javadoc

### CODE REVIEW

**Review by hisundar — APPROVED** [review]
LGTM

File: corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditor.java [hunk 1]
```diff
@@ -192,15 +194,16 @@ public int listTables(String namespace)
...
         String fullName = TableRegistry.getFullyQualifiedTableName(namespace, tablename);
         UUID streamUUID = UUID.nameUUIDFromBytes(fullName.getBytes());
         CorfuTable<CorfuDynamicKey, CorfuDynamicRecord> table =
             getTable(namespace, tablename);
         int tableSize = table.size();
-        log.info("Table {} in namespace {} with ID {} has {} entries",
-                tablename, namespace, streamUUID.toString(), tableSize);
-        log.info("\n======================\n");
+        System.out.println("Table " + tablename + " in namespace " + namespace +
+            "with ID " + streamUUID.toString() + " has " + tableSize + " " +
```
Comment (zfrenette) [inline corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditor.java:204]: Extra space here too.

**Review by zfrenette — APPROVED** [review]

File: corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditor.java [hunk 2]
```diff
@@ -242,20 +245,21 @@ public int printAllProtoDescriptors() {
...
         try {
             runtime.getObjectsView().TXBegin();
             CorfuTable<CorfuDynamicKey, CorfuDynamicRecord> table =
                 getTable(namespace, tablename);
             int tableSize = table.size();
-            log.info("Table {} in namespace {} with ID {} with {} entries will be dropped...",
-                tablename, namespace, streamUUID.toString(), tableSize);
+            System.out.println("Table " + tablename + " in namespace " + namespace
+                + "with ID " + streamUUID.toString() + " with " + tableSize +
+                "entries will be dropped...");
```
[the remark this answers is not in the document]
Comment (pankti-m) [reply corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditor.java:258]: it is the blank space between 2 strings

File: corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditor.java [hunk 3]
```diff
@@ -516,10 +524,10 @@ public void onError(Throwable throwable) {
             record -> streamTags.addAll(record.getMetadata()
                 .getTableOptions().getStreamTagList()));
 
-        log.info("\n======================\n");
-        log.info("Total unique stream tags: [{}]", streamTags.size());
-        streamTags.forEach(log::info);
-        log.info("\n======================\n");
+        System.out.println("\n======================\n");
+        System.out.println("Total unique stream tags: " + streamTags.size());
+        streamTags.forEach(tag -> System.out.println(tag));
```
[the remark this answers is not in the document]
Comment (pankti-m) [reply corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditor.java:529]: will add it in a follow-up change if I dont make further changes in this PR

File: corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditor.java [hunk 4]
```diff
@@ -607,6 +616,7 @@ private void printStreamTagMap(String tag, List<TableName> tables) {
         }
         // Remove last continuation characters for a clean output ', '
         formatMapping = formatMapping.substring(0, formatMapping.length() - 2);
-        log.info("tag: <'{}'> --- Tables[total={}] :: {{}}", tag, tables.size(), formatMapping);
+        System.out.println("Tag: " + tag + " --- Total Tables: " + tables.size()
+            + "TableNames: " + formatMapping);
```
[the remark this answers is not in the document]
Comment (pankti-m) [reply corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditor.java:620]: yes, let me make a note of it.  Thanks.

**Pull request merged**